docs/api/commands/origin.mdx

```diff
@@ -406,54 +406,32 @@ into the callback.
 
 ### Dependencies / Sharing Code
 
-[ES module dynamic `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
-and/or
+[`Cypress.require()`](/api/cypress-api/require) can be utilized to include
+[npm](https://www.npmjs.com/) packages and other files. It is functionally the
+same as using
 [CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
-can be used within the callback to include [npm](https://www.npmjs.com/)
-packages and other files.
+in browser-targeted code.
 
-:::caution
-
-Using `import()` and `require()` within the callback requires enabling the
-[`experimentalOriginDependencies`](/guides/references/experiments) flag in the
-Cypress configuration and using version `5.15.0` or greater of the
-[`@cypress/webpack-preprocessor`](https://github.com/cypress-io/cypress/tree/master/npm/webpack-preprocessor).
-The `@cypress/webpack-preprocessor` is included in Cypress by default, but if
-your project installs its own version in the Cypress configuration, make sure it
-is version `5.15.0` or greater.
-
-If using an older version of the webpack or a different preprocessor, you'll see
-an error that includes the following text:
-
-_Using `require()` or `import()` to include dependencies requires enabling the
-`experimentalOriginDependencies` flag and using the latest version of
-`@cypress/webpack-preprocessor`._
-
-:::
+Note that it is not possible to use
+[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+or
+[ES module `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports)
+within the callback.
 
 :::caution
 
-Using `require()` or `import()` within the callback from a `node_modules` plugin
-is not currently supported. We anticipate adding support with issue
-[#24976](https://github.com/cypress-io/cypress/issues/24976).
+Using `Cypress.require()` within the callback requires enabling the
+[`experimentalOriginDependencies`](/guides/references/experiments) flag in the
```

Comment:
```suggestion
[`experimentalOriginDependencies`](/guides/references/experiments) option in the
```

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/5020/commits/257a803e312e274f0e654dbe73ac7cc883a88d21